<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLotteryTicket</class>
 <widget class="QWidget" name="KLotteryTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KLotteryTicket</string>
  </property>
  <widget class="QLabel" name="label_ChosenNums">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择的号码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_InputNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入的号码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PrizeNums">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>中奖的号码：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_ChosenNums">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_InputNum">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_PrizeNums">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Money">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>奖金：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_Money">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton0">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>411</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开奖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearall">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>clear all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Reset">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
